doc/radosgw/lua-scripting.rst

```diff
@@ -511,3 +526,39 @@ in `postRequest` context, we can add attributes and events to the request's trac
 
   Request.Trace.AddEvent("second event", event_attrs)
 
+- Calculate the entropy and size of uploaded objects and print to debug log
```

Comment:
I'm curious, how is this entropy value useful?

Reply:
could be used to detect encryption of objects. added this to the doc